src/base/filesys_temp.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * file_sys.cpp: File System namespace & functionality
+ */
+#include "filesys_temp.hpp"
+#include <stdlib.h>
+
+#ifdef LINUX
+#include "porting.h"
+#elif WINDOWS
+#include <cstdio>
+#include <fcntl.h>
+#include <io.h>
+#endif
+
+namespace
+{
+  std::string unique_tmp_filename (const char *prefix="cub_") //generates an unique filename in tmp folder
+  {
+#ifdef LINUX
+    std::string filename = std::string ("/tmp/") + prefix + "XXXXXX"; //used with mkstemp()
```

Comment:
"/tmp" directory may not be available for security reasons.
So, It would be better to use the $TMPDIR as the tmpname() function instead of fixing it to "/tmp" directory.
If $TMPDIR doesn't exist, you will use "/tmp" directory.

Reply:
I changed to std::filesystem::temp_directory_path () to better handle the path

Reply:
OK. it sounds good.